Sources/NIO/Linux.swift

```diff
@@ -113,6 +113,10 @@ internal enum Epoll {
 }
 
 internal enum Linux {
+    static let CFS_QUOTA_PATH = "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
```

Comment:
It'd be quite nice to swiftify those a bit, maybe `cfsQuotaPath`, `cfsPeriodPath`, and `cpuSetPath`?